### ISSUE TITLE
New-LocalUser , PasswordRequired property

### ISSUE THREAD

**Issue by m4dm4n** [PowerShell/PowerShell#11965]
# Steps to reproduce

When creating a new local user via New-LocalUser cmdlet with password, property RequiredPassword gives value $false, although password IS required when attempting to logon.

```
New-LocalUser -Name "Test1" -Password $Password

Get-LocalUser Test1 | Format-List

AccountExpires         : 
Description            :
Enabled                : True
FullName               :
PasswordChangeableDate : 2/27/2020 12:25:25 PM
PasswordExpires        : 4/9/2020 1:25:25 PM
UserMayChangePassword  : True
**PasswordRequired       : False**
PasswordLastSet        : 2/27/2020 12:25:25 PM
LastLogon              :
Name                   : Test1
SID                    : S-1-5-21-2902974129-3397675831-214796114-1009
PrincipalSource        : Local
ObjectClass            : User

```

But, when creating new user via "net user" command, RequiredPassword property show expected $true value

```
net user Test2 * /ADD

Get-LocalUser Test2 | Format-List

AccountExpires         : 
Description            : 
Enabled                : True
FullName               : 
PasswordChangeableDate : 2/27/2020 12:27:16 PM
PasswordExpires        : 4/9/2020 1:27:16 PM
UserMayChangePassword  : True
**PasswordRequired       : True**
PasswordLastSet        : 2/27/2020 12:27:16 PM
LastLogon              : 
Name                   : Test2
SID                    : S-1-5-21-2902974129-3397675831-214796114-1010
PrincipalSource        : Local
ObjectClass            : User

```

# Expected behavior

After creating a user, property PasswordRequired should be true.

# Actual behavior

After creating a user, property PasswordRequired is false.

# Additional data

You can clearly see that this property has a set option, but there is no way to actually set it to $true or $false.

```
Get-LocalUser | Get-Member -Name PasswordRequired                 

   TypeName: Microsoft.PowerShell.Commands.LocalUser
Name             MemberType Definition
----             ---------- ----------
PasswordRequired Property   bool PasswordRequired {get;set;}

```

So basically, when creating a user via New-LocalUser with password, property PasswordRequired should automatically be set to True value. 

# Environment data

```

Name                           Value
----                           -----
PSVersion                      6.2.4
PSEdition                      Core
GitCommitId                    6.2.4
OS                             Microsoft Windows 10.0.18362
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by jhoneill** [comment]
IIRC if "require password" is false the user can set an empty password.  It is not "currently has an empty password".  

I can't offer a reason why one tool forbids local users to set null passwords and the other allows it. Possibly because it should be set by a global policy not user-by-user.

**Comment by m4dm4n** [comment]
Yes, but actually no.

```
net user Test3 /add

Get-LocalUser Test3 | Format-List

AccountExpires         :
Description            :
Enabled                : True
FullName               :
PasswordChangeableDate :
PasswordExpires        : 4/10/2020 9:38:01 AM
UserMayChangePassword  : True
**PasswordRequired       : True**
PasswordLastSet        : 2/28/2020 8:38:01 AM
LastLogon              :
Name                   : Test3
SID                    : S-1-5-21-2902974129-3397675831-214796114-1011
PrincipalSource        : Local
ObjectClass            : User

```

On the first logon, I didn't have to add any password. So it's not really an requirement, nor an information, I think something somewhere is broken.

**Comment by jhoneill** [comment]
If you check the user in the MMC does that say they require a password ? It seems like the issue might be in the net command (which goes back to the mid 1980s) :-)

**Comment by iSazonov** [comment]
The cmdlet is not in the repo.

**Comment by m4dm4n** [comment]
> 
> 
> The cmdlet is not in the repo.

https://github.com/PowerShell/PowerShell/tree/master/src/Microsoft.PowerShell.LocalAccounts/LocalAccounts/Commands

**Comment by iSazonov** [comment]
@m4dm4n The module was removed by compliance reasons.

**Comment by iSazonov** [comment]
Fixed in community module https://www.powershellgallery.com/packages/LocalAccounts
